templates/common/root/readme.md

```diff
@@ -0,0 +1,151 @@
+# <%= _.slugify(_.humanize(appname)) %>
+
+This project is generated with [yo angular generator](https://github.com/yeoman/generator-angular).
+
+Run `grunt` for building and `grunt serve` for preview.
+
+## Generators
```

Comment:
I don't know if the generators section is actually appropriate for the projects readme after all. It doesn't help describe the project and if the project only used Yeoman to generate it and not maintain it then its instructions that don't help.